src/errors.js

```diff
@@ -50,7 +50,7 @@ export class NotFoundError extends GlobalError {
 		}, options))
 	}
 	get detail() {
-		return `No ${this.type} was found with the specified ID.`
+		return `No ${this.type} was found with the specified ID.  If this report was generated by live logging, it may be cached -- try pressing Refresh to reload it.`
```

Comment:
```suggestion
		return `No ${this.type} was found with the specified ID. If this report was generated by live logging, it may have been cached - try pressing Refresh to reload it.`
```